ra/ra_test.go

```diff
@@ -310,9 +348,11 @@ func initAuthorities(t *testing.T) (*DummyValidationAuthority, sapb.StorageAutho
 
 	saDBCleanUp := test.ResetBoulderTestDatabase(t)
 
-	va := &DummyValidationAuthority{
-		performValidationRequest: make(chan *vapb.PerformValidationRequest, 1),
+	dummyVA := &DummyValidationAuthority{
+		doDCVRequest: make(chan *vapb.PerformValidationRequest, 2),
```

Comment:
why does the buffer size of this channel need to increase?

Reply:
It actually doesn't. Turns out I was compensating for the extra `dva.doDCVRequest <- req` inside of `func (dva *DummyValidationAuthority) PerformValidation(ctx context.Context, req *vapb.PerformValidationRequest, _ ...grpc.CallOption)`